stmhal/boards/CERB40/mpconfigboard.h

```diff
@@ -16,6 +16,7 @@
 #define MICROPY_HW_ENABLE_SERVO     (0)
 #define MICROPY_HW_ENABLE_DAC       (1)
 #define MICROPY_HW_ENABLE_CAN       (1)
+#define MICROPY_PY_SOFTIRQ          (1)
```

Comment:
You don't need these in all the board configs.  Just put it in stmhal/mpconfigport.h and it'll be enabled for all boards.